### PR TITLE
fixed the distribution plot

### PR DESCRIPTION
**Pull request by bluest93**
# Change
1. Fixed the distribution plot in the `graph_partition` notebook so that the frequency of the sample with the same energy will be added up
2. Changed the title
3. Fixed the issue in showing the equation in the markdown

# Description 
Used the dict method by importing `from collections import defaultdict`

### CODE REVIEW

**Comment by bluest93** [comment]
@Jacomichi Updated the code of distribution code

File: docs/tutorial/graph_partition.ipynb [hunk 1]
```diff
@@ -910,14 +931,18 @@
...
+    "\n",
+    "# Initialize a dictionary to accumulate occurrences for each energy value\n",
+    "frequencies = defaultdict(int)\n",
+    "\n",
+    "# Define the precision to which you want to round the energy values\n",
+    "for sample in sampleset:\n",
+    "    energy = round(sample.eval.objective, ndigits = 3)  \n",
+    "    frequencies[energy] += sample.num_occurrences\n",
+    "\n",
+    "plt.bar(frequencies.keys(), frequencies.values(), width=0.05)\n",
```
Comment (Jacomichi) [inline docs/tutorial/graph_partition.ipynb:943]: I think in this problem `width = 0.05` is too narrow, so can you change `width = 0.5`?

**Review by Jacomichi — CHANGES_REQUESTED** [review]
I left tiny comment

File: docs/tutorial/graph_partition.ipynb (hunk 1 above)
Comment (bluest93) [reply docs/tutorial/graph_partition.ipynb]: @Jacomichi Done

**Comment by bluest93** [comment]
@Jacomichi Done

**Review by Jacomichi — APPROVED** [review]
LGTM

**Pull request merged**